packages/duf/README.md

```diff
@@ -0,0 +1,84 @@
+<p align="center">
+    <a href="https://seequent.com" target="_blank">
+        <picture>
+            <source media="(prefers-color-scheme: dark)"
+                srcset="https://developer.seequent.com/img/seequent-logo-dark.svg"
+                alt="Seequent logo" width="400" />
+            <img src="https://developer.seequent.com/img/seequent-logo.svg" alt="Seequent logo" width="400" />
+        </picture>
+    </a>
+</p>
+<p align="center">
+    <a href="https://pypi.org/project/evo-data-converters-duf/">
+        <img alt="PyPI - Version" src="https://img.shields.io/pypi/v/evo-data-converters-duf" />
+    </a>
+    <a href="https://github.com/SeequentEvo/evo-data-converters/actions/workflows/on-merge.yaml">
+        <img src="https://github.com/SeequentEvo/evo-data-converters/actions/workflows/on-merge.yaml/badge.svg" alt=""/>
+    </a>
+</p>
+<p align="center">
+    <a href="https://developer.seequent.com/" target="_blank">Seequent Developer Portal</a>
+    &bull; <a href="https://community.seequent.com/" target="_blank">Seequent Community</a>
+    &bull; <a href="https://seequent.com" target="_blank">Seequent website</a>
+</p>
+
+## Evo
+
+Evo is a unified platform for geoscience teams. It enables access, connection, computation, and management of subsurface
+data. This empowers better decision-making, simplified collaboration, and accelerated innovation. Evo is built on open
+APIs, allowing developers to build custom integrations and applications. Our open schemas, code examples, and SDK are
+available for the community to use and extend. 
+
+Evo is powered by Seequent, a Bentley organisation.
+
+## Pre-requisites
+
+* Python virtual environment with Python 3.10, 3.11, or 3.12
+* Git
+* Deswik Suite
+
+## Installation
+
+To do.
+
+## DUF
+
+Deswik Unified File (DUF) is a proprietary file format from Deswik Mining Consultants Pty Ltd.
+
+### Publish geoscience objects from a DUF file
+[The `evo-data-converters-common` library](packages/common/README.md) can be used to sign in. After successfully signing
```

Comment:
```suggestion
[The `evo-data-converters-common` library](https://github.com/SeequentEvo/evo-data-converters/blob/main/packages/common/README.md) can be used to sign in. After successfully signing
```

I think this would be better - we have a process that transforms relative links into absolute links for PyPI (because PyPI doesn't do that itself). It probably won't work across packages within the monorepo, so we should either use relative links to the GitHub repo, or to the PyPI package.